src/test/java/com/exasol/udfdebugging/modules/ExampleUdfSetup.java

```diff
@@ -0,0 +1,57 @@
+package com.exasol.udfdebugging.modules;
+
+import java.io.Closeable;
+import java.sql.*;
+import java.util.stream.Stream;
+
+import com.exasol.bucketfs.Bucket;
+import com.exasol.containers.ExasolContainer;
+import com.exasol.dbbuilder.dialects.exasol.*;
+import com.exasol.dbbuilder.dialects.exasol.udf.UdfScript;
+import com.exasol.udfdebugging.ExposedServiceAddress;
+import com.exasol.udfdebugging.LocalServiceExposer;
+
+public class ExampleUdfSetup implements Closeable, AutoCloseable {
+    private static final String SCHEMA_NAME = "TEST";
+    private static final String UDF_NAME = "HELLO_WORLD";
+
+    private final ExasolContainer<? extends ExasolContainer<?>> exasol = new ExasolContainer<>().withReuse(true);
+
+    public ExampleUdfSetup() {
+        this.exasol.start();
+    }
+
+    public LocalServiceExposer getHostPortProxy() {
+        return port -> new ExposedServiceAddress(this.exasol.getHostIp(), port);
+    }
+
+    public Bucket getDefaultBucket() {
+        return this.exasol.getDefaultBucket();
+    }
+
+    public ExasolContainer<? extends ExasolContainer<?>> getExasolContainer() {
```

Comment:
This method is not used, right?

Reply:
not in the committed code. But I've prepared some other tests that use it